docs/assets/fhir-resource-ingestion-digestion-diagrams/bfd-pipeline-ccw-ingestion.plantuml

```diff
@@ -0,0 +1,27 @@
+@startuml
+title BFD-Pipeline-CCW Ingestion of Beneficiary/Claim Data Sequence
+participant ccw
+database s3
+participant slack
+participant "bfd-pipeline"
+database "bfd-db"
+ccw -> s3: Timestamped folder with: \n RIF files for a resource, manifest.xml \n placed in 'bfd-prod-etl' S3 bucket  
+s3 -> s3: Timestamped folder moved to \n 'Incoming' folder of bucket
+s3 -> slack: Data discovery status sent \n to #BFD-notices channel
+slack --> s3
+s3 -> "bfd-pipeline": ETL/RIF Loader process triggered
+group ETL
+"bfd-pipeline" -> "bfd-pipeline": datasetMonitorListener interface instantiated \n in CcwRifLoadJob class to manage ETL
+"bfd-pipeline" -> "bfd-pipeline": ETL process parses manifest.xml file \n for specific resource data \n and checks for DB operation \n denoted  in manifest i.e. INSERT/UPDATE
+"bfd-pipeline" -> "bfd-db": DB operations applied for resource \n based on column and rif layout fhir mappings
+"bfd-db" --> "bfd-pipeline"
+"bfd-pipeline" -> "bfd-pipeline": Process marked as 'complete'
+end
+"bfd-pipeline" -> s3: Load status sent to S3
+s3 -> slack: Status sent \n to #BFD-notices channel
```

Comment:
Perhaps this is aspirational and how it _should_ work, but today...
* the bfd-pipeline service produces logs which are scraped and forwarded by splunk
* a scheduled splunk query runs every 15 minutes to find specific discovery/load messages for the previous 15 minutes
* splunk formats messages based on those indications of discovery/load to #bfd-notices

Make it your own, but, I think we can afford to be a _little_ lossier here and add some of the following:

* `participant splunk` to represent a sort of _composite_ participant representing e.g. splunk's various components of forwarders/indexers/search heads/ui/configuration
* `"bfd-pipeline" --> splunk: forwarded service logs`
* `splunk --> slack: discovery`
* `splunk --> slack: load`

Reply:
Added